samples/snippets/requirements.txt

```diff
@@ -0,0 +1,9 @@
+db-dtypes==1.2.0
+google.cloud.bigquery==3.19.0
+google-cloud-bigquery-storage==2.24.0
+ipython===7.31.1; python_version == '3.7'
+ipython===8.0.1; python_version == '3.8'
+ipython===8.18.1; python_version >= '3.9'
+pandas==1.3.5; python_version == '3.7'
+pandas==2.0.0; python_version == '3.8'
+pandas==2.2.1; python_version >= '3.9'
```

Comment:
Ideally this will just be `bigquery_magics` with no extra dependencies. See this comment: https://github.com/googleapis/python-bigquery/blob/bf8861c3473a1af978db7a06463ddc0bad86f326/samples/snippets/requirements.txt#L1